build.sbt

```diff
@@ -62,7 +85,13 @@ lazy val rainierCore = project.
   in(file("rainier-core")).
   settings(name := "rainier-core").
   dependsOn(shadedAsm).
-  settings(commonSettings)
+  settings(commonSettings).
+  settings(
+    bazelRuleDeps := (Deps(Compile) - ScalaLib(Compile)) +
+      BazelDep("//.rainier-shaded-asm", "asmTreeShaded") +
```

Comment:
This manually adds the two shaded targets that are created below. See the `shadedAsm` project.